### PR TITLE
az/mr/lw 5pm-1-Redirects user to add schedule page after clicking on 'Add' button

### PR DESCRIPTION
**Pull request by alanzhang052**
Users are now redirected to an "Add To Schedule" page to determine which schedule they want to add a course into.

### CODE REVIEW

**Review by evantyip — DISMISSED** [review]
LGTM, works on localhost and passes all tests

File: javascript/src/main/components/Schedule/AddToScheduleTable.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+import BootstrapTable from 'react-bootstrap-table-next';
+import { Button } from "react-bootstrap";
+
+const AddToScheduleTable = ({ data }) => {
+
+  const renderAddToButton = (id) => {
+     return (
+       <Button data-testid={`add-to-button-${id}`} onClick={() => {
+         
+       }}>Add To</Button>
```
Comment (andrewhlu) [inline javascript/src/main/components/Schedule/AddToScheduleTable.js:11]: If the `onClick` function is empty, can we just remove it?

File: javascript/src/main/App.js [hunk 2]
```diff
@@ -31,6 +31,7 @@ import AdminSettings from "main/pages/Admin/AdminSettings";
 import useSWR from "swr";
 import { fetchWithToken } from "main/utils/fetch";
 import CourseName from "./pages/History/CourseName";
+import AddToSchedule from "main/pages/Schedule/AddToSchedule"
```
Comment (andrewhlu) [inline javascript/src/main/App.js:34]:
```suggestion
import AddToSchedule from "main/pages/Schedule/AddToSchedule";
```

**Review by andrewhlu — CHANGES_REQUESTED** [review]
Not a full review (haven't tested on QA), but noticed some small things to fix:

File: javascript/src/main/components/Schedule/AddToScheduleTable.js (hunk 1 above)
Comment (alanzhang052) [reply javascript/src/main/components/Schedule/AddToScheduleTable.js:11]: We were planning on implementing the function later in another issue, but you're right, we can remove it until next issue

File: javascript/src/main/components/Schedule/AddToScheduleTable.js [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+  const renderAddToButton = (id) => {
+     return (
+       <Button data-testid={`add-to-button-${id}`}>Add To</Button>
+     )
+  }
+
+  const columns = [{
+    dataField: 'id',
+    text: 'id'
+  }, {
```
Comment (andrewhlu) [inline javascript/src/main/components/Schedule/AddToScheduleTable.js:16]: I am slightly bothered by the fact that this line isn't consistent with the rest of this array lol
It's up to you if you would like to fix it

Comment (lwu5) [reply javascript/src/main/components/Schedule/AddToScheduleTable.js]: fixed!

**Review by evantyip — APPROVED** [review]
LGTM

**Pull request merged**